packages/eddsa-frog-pcd-ui/package.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
One thing to test before merging is whether a clean repo builds and publishes properly to your local NPM, so that we prevent issues like the ones we encountered the last time we did significant packaging changes.

Reply:
It does - the packages all build and publish properly. The immediate motivation for this change was to fix an issue with a third-party codebase, so in order to test that the fix works I had to build all of the packages locally and test the third-party project building against the new versions.